doc/Analytics.xml

```diff
@@ -3373,12 +3374,8 @@ Added Recognition rule engines</revremark>
         language:</para>
       <programlisting><![CDATA[<tt:RuleDescription Name="tt:LoiteringDetector">
   <tt:Parameters>
-    <tt:SimpleItemDescription  Name="TimeThreshold"
-      Type="xs:nonNegativeInteger"/>
-    <tt:SimpleItemDescription  Name="isPeriodicNotify" Type="xs:boolean"/>
-    <tt:SimpleItemDescription  Name="TimeInterval"     Type="xs:duration"/>
-    <tt:ElementItemDescription Name="Field"
-      Type="tt:PolygonConfiguration"/>
+    <tt:SimpleItemDescription  Name="TimeThreshold" Type="xs:int"/>
```

Comment:
Can we have this TimeThreshold as xs:duration ?

Reply:
Good idea - will update